examples/src/routes/AttributeFilter.jsx

```diff
@@ -26,6 +28,17 @@ export const AttributeFilter = () => (
             Pass a custom onApply function to this component to handle what happens when the user clicks the
             Apply button.
         </p>
+        <ExampleWithSource
+            for={AttributeFilterComponentExample}
+            source={AttributeFilterComponentExampleSRC}
+        />
+
+        <hr className="separator" />
+
+        <h2>Attribute Filter Example</h2>
+
+        <p>This example shows how to apply attribute filter component into report.</p>
```

Comment:
`This example shows how to add attribute filter component into a report.` or `This example shows how to apply attribute filter component on the report.`